charts/minecraft/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft
-version: 3.1.7
+version: 3.1.10
```

Comment:
I think you'll only need to bump the version once for the overall PR

Reply:
doh!